test_cases/encoding.json

```diff
@@ -44,6 +44,48 @@
           }
         ]
       }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "endpoint": "search",
+      "description": "the Chambéry accent issue, with the accented e represented as 1 character",
```

Comment:
Now that you get github emails you will see this message referring to my previous comment which says "Adding a link to the original issue (pelias/api#600) would be great here and in the next test"